### ISSUE TITLE
Too little contrast in progress indicator of inactive tab in light theme

### ISSUE THREAD

**Issue by KalleOlaviNiemitalo** [microsoft/terminal#17475]
### Windows Terminal version

1.20.11381.0

### Windows build number

10.0.19045.4529

### Steps to reproduce

Configure the terminal to use the light theme (not legacy), and a tab bar separate from the title bar.
Open a PowerShell tab and run a command to output the OSC that sets 50% progress: ``"`e]9;4;1;50`e\"``
Do the same in another tab.

### Expected Behavior

The progress indicators in the tab bar should have good contrast so that I can see how much progress has been made.

### Actual Behavior

The progress indicator of the inactive tab is too light and doesn't have enough contrast to the light background:  
![image](https://github.com/microsoft/terminal/assets/46201428/5e68fa33-16fc-44ea-8a8c-c61c0884af3a)

Compare to the dark theme, where both progress indicators have adequate contrast:
![image](https://github.com/microsoft/terminal/assets/46201428/407b68f2-fb8c-474c-80e0-6ab7bddbcb51)

**Comment by zadjii-msft** [comment]
I have this gut feeling that this was fixed in Windows 11:
![image](https://github.com/microsoft/terminal/assets/18356694/ae28cea3-6d04-49e2-8e0f-6fca789d0f05)
![image](https://github.com/microsoft/terminal/assets/18356694/511c62e5-e409-4e1a-a812-f1c94d8cb4d6)
![image](https://github.com/microsoft/terminal/assets/18356694/a3f54283-b1a0-40d0-8eb2-e14ff324b31c)


I bet the accent color logic got updated to be smarter in Win11

**Comment by zadjii-msft** [comment]
Yea, it looks like there was a lot of work in this area in Windows 11. Clearly these aren't all publicly accessible issue threads, but there is a lot of discussion in the win11 timeframe about accent colors:

* MSFT:25681566 
* MSFT:30833850
* MSFT:26006362
* https://github.com/microsoft/microsoft-ui-xaml/issues/2354

I'm guessing that a backport to Win10 is highly unlikely. I have no idea how challenging this would be to workaround - possibly prohibitively hard. I don't think WinUI 2 lets us easily restyle the progress ring.

**Comment by zadjii-msft** [comment]
FWIW, I was using the WinUI 2 gallery for those screenshots 😉

----

I'm thinking:
* we've already got code that checks what the _evaluated_ color of a tab is
* we've got code to swap out the resources in the tab
* presumably, we can swap out those resource for the progress ring too?
* so theoretically if someone has a light tab (either light theme or a dark theme + white titlebar) we can always manually replace it with the dark accent color (and vice versa)
